src/arch/xtensa/configs/apollolake_defconfig

```diff
@@ -3,3 +3,4 @@ CONFIG_CAVS_DMIC=y
 CONFIG_CAVS_SSP=y
 CONFIG_LP_MEMORY_BANKS=2
 CONFIG_HP_MEMORY_BANKS=8
+CONFIG_PERFORMANCE_COUNTERS=y
```

Comment:
@mmaka1 Any reason this option is added for APL but not for ICL/TGL ?

Reply:
I have been observing CNL and APL specifically. If you have another platform to troubleshoot pls turn it on.

Reply:
Shouldn't it be something that is  enabled in  custom-debug build? Doesn't look like config that should be in defconfig.